Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/project.json

```diff
@@ -1,11 +1,11 @@
 ﻿{
   "dependencies": {
-    "Xamarin.Forms": "2.3.4.231",
-    "NETStandard.Library": "1.6.1"
+    "NETStandard.Library": "1.6.1",
+    "Xamarin.Forms": "2.4.0.18342"
   },
   "frameworks": {
-      "netstandard1.0": {
-        "imports": "portable-net45+win8+wpa81"
+    "netstandard1.0": {
+      "imports": "portable-net45+win8+wpa81"
```

Comment:
Is the fallback still needed with XF 2.4.0?

Reply:
I guess not, will check later as develop branch has an issue with UWP.